pylint/checkers/variables.py

```diff
@@ -405,7 +405,11 @@ def _has_locals_call_after_node(stmt, scope):
     "E0601": (
         "Using variable %r before assignment",
         "used-before-assignment",
-        "Used when a local variable is accessed before its assignment.",
+        "Emitted when a local variable is accessed before its assignment took place. "
+        "Assignments in try blocks are assumed not to have occurred when evaluating "
+        "associated except/finally blocks. Assignments in except blocks are assumed "
+        "not to have occurred, except when the associated try block contains a return "
```

Comment:
This last part is not actually true for this PR right? Might as well remove it and keep the git history accurate.